CRM/Core/ManagedEntities.php

```diff
@@ -421,6 +433,10 @@ protected function removeStaleEntity($dao) {
         $doDelete = FALSE;
         break;
 
+      case 'unmodified':
+        $doDelete = empty($dao->entity_modified_date);
```

Comment:
> > If you use attempt to use the new policy on an entity that doesn't support it, what's expected to happen? Does it effectively use some other mode? Or does it raise an error?
>
> The former. Because the modification timestamp is only updated for supported entities, for all others it will remain null; which in practice would cause the mode to act like "always".

Would this be a sensible place to emit a warning if someone has mixed `unmodified` with an entity that lacks modification-tracking?

I like that it's able to execute and without crashing. :+1:  It seems support would roll out incrementally (entity-by-entity/version-by-version) - so we'd expect various permutations {extension-version x managed-entity-policy x core-version x ...} over time. If a particular configuration declares a policy that can't be met, then a warning seems appropriate...